tests/unit/mockData/mockDataBuilder.js

```diff
@@ -0,0 +1,8 @@
+const path = require('path');
+const fs = require('fs');
+
+export const getMockedFileBuffer = (filename) => {
+  // TODO: review this method
```

Comment:
TODO?

Reply:
We added this TODO because reading the file could fail, and this would cause all the tests failing for the wrong reasons. The problem was that we couldn't find out how to mock the buffer with the data. Do you think we should keep trying? Or should we remove the TODO and expect that the function will always work?

Reply:
Seems unlikely the unittest would fail to read the file. Maybe add an assert the file is read correctly if that is easily doable but otherwise just document what you said and leave as-is